js/views/cfi_navigation_logic.js

```diff
@@ -2,166 +2,184 @@
 //
 //  Created by Boris Schneiderman.
 //  Copyright (c) 2014 Readium Foundation and/or its licensees. All rights reserved.
-//  
```

Comment:
Code auto-format should not be committed in this PR (if there is a significant and PR-relevant code change, we need to narrow it down to a palatable number of lines).